coding-standards.md

```diff
@@ -284,4 +284,31 @@ public static Array BuildObjectArray(Type type, IEnumerable list, bool distinct)
     newList.CopyTo(a, 0);
     return a;
 }
-```
\ No newline at end of file
+```
+
+## Easy PR's		
+
+Approach you're change responsibly. Do not submit massive PR's. 
+
+[pull requests](https://help.github.com/articles/about-pull-requests/) on GitHub. 		
+		
+## Build and Tests		
+		
+There are a few aspects to the build that requires a little setup and further exploration. 		
+		
+ - Visual Studio needs to be run in `admin` mode.		
```

Comment:
I think we've got some wires crossed, I don't think Windsor specific details belong in here especially that'll it'll confuse anyone working on other Castle projects.

Do you think all of this is needed if we implement https://github.com/castleproject/Windsor/issues/174?